[2025-02-10T15:26:05.927+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-10T15:26:06.050+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.verify_file_reliability scheduled__2025-02-10T15:25:00+00:00 [queued]>
[2025-02-10T15:26:06.059+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.verify_file_reliability scheduled__2025-02-10T15:25:00+00:00 [queued]>
[2025-02-10T15:26:06.060+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-10T15:26:06.072+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): verify_file_reliability> on 2025-02-10 15:25:00+00:00
[2025-02-10T15:26:06.078+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=6759) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-10T15:26:06.080+0000] {standard_task_runner.py:72} INFO - Started process 6761 to run task
[2025-02-10T15:26:06.081+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag', 'verify_file_reliability', 'scheduled__2025-02-10T15:25:00+00:00', '--job-id', '343', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmpc81792mw']
[2025-02-10T15:26:06.082+0000] {standard_task_runner.py:105} INFO - Job 343: Subtask verify_file_reliability
[2025-02-10T15:26:06.123+0000] {task_command.py:467} INFO - Running <TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.verify_file_reliability scheduled__2025-02-10T15:25:00+00:00 [running]> on host b6443be51f35
[2025-02-10T15:26:06.214+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag' AIRFLOW_CTX_TASK_ID='verify_file_reliability' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T15:25:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-10T15:25:00+00:00'
[2025-02-10T15:26:06.215+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-10T15:26:06.216+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-10T15:26:06.217+0000] {logging_mixin.py:190} INFO - Current task name:verify_file_reliability state:running start_date:2025-02-10 15:26:06.051761+00:00
[2025-02-10T15:26:06.217+0000] {logging_mixin.py:190} INFO - Dag name:IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag and current dag run status:running
[2025-02-10T15:26:06.218+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-10T15:26:06.219+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-10T15:26:06.220+0000] {logging_mixin.py:190} INFO - camarche22 /opt/***/files/in ['.csv'] ['utf-8', 'ISO-8859-1', 'Windows-1252']
[2025-02-10T15:26:06.231+0000] {my_directory_and_file_checker.py:241} ERROR - ❌ Erreur d'encodage (utf-8) pour Écritures comptables article (10).csv: 'utf-8' codec can't decode byte 0xb0 in position 1: invalid start byte
[2025-02-10T15:26:06.264+0000] {logging_mixin.py:190} INFO - 
📂 Contenu du fichier (ISO-8859-1): Écritures comptables article (10).csv
[2025-02-10T15:26:06.274+0000] {logging_mixin.py:190} INFO -   Date comptabilisation      N° lot Type écriture N° document  N° séquence
0            16/12/2024         NaN         Vente   BL1900145        15735
1            16/12/2024         NaN         Vente      107012        15734
2            11/01/2025  16LABCDE24    Production   RPO000460        15733
3            11/01/2025  16LABCDE24    Production   RPO000460        15732
4            11/01/2025  16LABCDE24    Production   RPO000460        15731
[2025-02-10T15:26:07.217+0000] {logging_mixin.py:190} INFO - ❌ Erreur Ligne 10923 : accounting_date - nan -> Type attendu : date
[2025-02-10T15:26:07.241+0000] {logging_mixin.py:190} INFO - ✅Lignes  ('✅ Le fichier respecte les colonnes attendues et les types de données.', False) 10924
[2025-02-10T15:26:07.245+0000] {logging_mixin.py:190} INFO - Merde
[2025-02-10T15:26:07.248+0000] {python.py:240} INFO - Done. Returned value was: end
[2025-02-10T15:26:07.249+0000] {branch.py:38} INFO - Branch into end
[2025-02-10T15:26:07.249+0000] {skipmixin.py:230} INFO - Following branch ('end',)
[2025-02-10T15:26:07.258+0000] {skipmixin.py:278} INFO - Skipping tasks [('import_temp_ecc_to_ecc', -1)]
[2025-02-10T15:26:07.288+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-10T15:26:07.289+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag, task_id=verify_file_reliability, run_id=scheduled__2025-02-10T15:25:00+00:00, execution_date=20250210T152500, start_date=20250210T152606, end_date=20250210T152607
[2025-02-10T15:26:07.299+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-10T15:26:07.300+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-10T15:26:07.300+0000] {logging_mixin.py:190} INFO - Dag name:IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag queued_at:2025-02-10 15:26:00.172931+00:00
[2025-02-10T15:26:07.300+0000] {logging_mixin.py:190} INFO - Task hostname:b6443be51f35 operator:BranchPythonOperator
[2025-02-10T15:26:07.340+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-10T15:26:07.366+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-10T15:26:07.371+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
